apps/src/templates/rubrics/LearningGoal.jsx

```diff
@@ -38,6 +58,87 @@ export default function LearningGoal({
     setIsOpen(!isOpen);
   };
 
+  const handleFeedbackChange = event => {
+    if (studentLevelInfo.user_id && learningGoal.id) {
+      if (autosaveTimer.current) {
+        clearTimeout(autosaveTimer.current);
+      }
+      teacherFeedback.current = event.target.value;
+      setDisplayFeedback(teacherFeedback.current);
+      autosaveTimer.current = setTimeout(() => {
```

Comment:
Could any of this be simplified by using [lodash's](https://lodash.com/docs/4.17.15#debounce) `debounce` method?

Reply:
That might save some lines. I'll check it out. Thanks!

Reply:
After looking into it, I found that using debounce wouldn't reduce the amount of code. It takes the same amount to set up debounce (plus importing the library), and I only run the timer once in the code.